### ISSUE TITLE
Client doesn't start with "minConnsPerNode" option

### ISSUE THREAD

**Issue by Aconradty** [aerospike/aerospike-client-nodejs#419]
I'm trying to use the `minConnsPerNode` setting but it doesn't seem to work.

```javascript
const Aerospike = require('aerospike')

const config = {
  hosts: '127.0.0.1:3000',
  minConnsPerNode: 1, // works if minConnsPerNode is removed
}

const run = async () => {
  Aerospike.setDefaultLogging({ level: Aerospike.log.DEBUG, file: process.stderr.fd })
  const client = await Aerospike.connect(config);

  const puts = Array.from({ length: 5 }, (_, i) => client.put(new Aerospike.Key('ssd-store', 'test', i), { i }))

  await Promise.all(puts)
  client.close()
}

run()

```

Console output without `minConnsPerNode`:

```
$ node index.js 
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [config.cc:241] [config_from_jsobject] - Parsing global policies: success
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [config.cc:329] [config_from_jsobject] - Built as_config instance from JS config object
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [client.cc:80] [New] - Aerospike client initialized successfully
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [as_pipe.c:276] [get_buffer_size] - Buffer limit is 212992, should be at least 5242880 if async pipelining is used. Please set /proc/sys/net/core/wmem_max accordingly.
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [as_pipe.c:276] [get_buffer_size] - Buffer limit is 212992, should be at least 15728640 if async pipelining is used. Please set /proc/sys/net/core/rmem_max accordingly.
Jun 02 2021 12:15:58 GMT: INFO (85204) [as_cluster.c:195] [as_cluster_add_nodes_copy] - Add node BB9030010AC4202 127.0.0.1:9090
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [events.cc:56] [push] - Cluster event 0 triggered by node "BB9030010AC4202" (127.0.0.1:9090)
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [as_node.c:890] [as_node_process_response] - Node BB9030010AC4202 peers generation changed: 0
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [as_node.c:897] [as_node_process_response] - Node BB9030010AC4202 partition generation changed: 0
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [client.cc:99] [Connect] - Successfully connected to cluster: Enjoy your cake!
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [command.cc:66] [Callback] - Executing JS callback for Connect command
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [put_async.cc:73] [PutAsync] - Sending async put command
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [put_async.cc:73] [PutAsync] - Sending async put command
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [put_async.cc:73] [PutAsync] - Sending async put command
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [put_async.cc:73] [PutAsync] - Sending async put command
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [put_async.cc:73] [PutAsync] - Sending async put command
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [command.cc:66] [Callback] - Executing JS callback for Put command
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [command.cc:66] [Callback] - Executing JS callback for Put command
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [command.cc:66] [Callback] - Executing JS callback for Put command
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [command.cc:66] [Callback] - Executing JS callback for Put command
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [command.cc:66] [Callback] - Executing JS callback for Put command
Jun 02 2021 12:15:58 GMT: DEBUG(85204) [client.cc:114] [Close] - Closing the connection to aerospike cluster
```

Console output with `minConnsPerNode`:

```
$ node index.js 
Jun 02 2021 12:16:46 GMT: DEBUG(85275) [config.cc:241] [config_from_jsobject] - Parsing global policies: success
Jun 02 2021 12:16:46 GMT: DEBUG(85275) [config.cc:329] [config_from_jsobject] - Built as_config instance from JS config object
Jun 02 2021 12:16:46 GMT: DEBUG(85275) [client.cc:80] [New] - Aerospike client initialized successfully
Jun 02 2021 12:16:46 GMT: DEBUG(85275) [as_pipe.c:276] [get_buffer_size] - Buffer limit is 212992, should be at least 5242880 if async pipelining is used. Please set /proc/sys/net/core/wmem_max accordingly.
Jun 02 2021 12:16:46 GMT: DEBUG(85275) [as_pipe.c:276] [get_buffer_size] - Buffer limit is 212992, should be at least 15728640 if async pipelining is used. Please set /proc/sys/net/core/rmem_max accordingly.
```

Tested with:
Node Version: v14.17.0
Aerospike Client Version: 3.16.5 & 3.16.0
Server Version: Aerospike Community Edition build 5.5.0.3 & Aerospike Enterprise Edition build 5.4.0.2

**Comment by jhecking** [comment]
Sorry for the very late response! I'm able to reproduce the issue. However, since the Node.js client just passes through the `minConnsPerNode` setting to the C client SDK, which handles all the connection management, I'll need help from our C client developer to look into the issue.

**Comment by jhecking** [comment]
Just a quick update: We've identified a potential fix in the Aerospike C client which looks promising. Once a new 4.6.x C client release is available I will release a new Node.js client version as well.

**Comment by jhecking** [comment]
This has been fixed in [aerospike@3.16.6](https://www.npmjs.com/package/aerospike).
